### ISSUE TITLE
Status of this library?

### ISSUE THREAD

**Issue by arnuschky** [shuoli84/gevent_socketio2#3]
Just discovered this project - seems great, as the original gevent-socketio is dead. What is the status of this project? Is it usable already? Anyone using it in production?

**Comment by shuoli84** [comment]
1. The project is "dead" too if no one interested, just like any other OS project.
2. I used it in my prev startup, but not battle tested.
3. No, now no one is using it. 

Thanks for asking, if you interested, we can bring it back to life, I've put a lot effort implement it. :)

**Comment by arnuschky** [comment]
Thank you for your quick reply. We're currently using the original gevent-socketio, and looking for an option to get away from this. Not sure at the moment whether we will stay with socketio or if we'll migrate to sockJS. If I have the time, I'll evaluate gevent_socketio2 over the next week. Can you summarize the differences to the original? Is it really a complete rewrite? How are you handling the "multiple worker" issue?

There are regularly people on the original gevent-socketio, so the interest is there. Maybe we could advertise this project a bit more?

**Comment by jdeschenes** [comment]
I was currently working on my own implementation of socketio-1.0 before I found this. I will merge my code with what you have already in the following days.

**Comment by shuoli84** [comment]
@jdeschenes  Sorry for the late reply. This project is dead. If you think any code useful, go ahead and use it. Due to many reasons, I switched to Java netty socketio.

**Comment by gkadillak** [comment]
We're currently using this code for the socketio 1.x implementation. I've created my own fork and plan on maintaining the project. Come support the effort https://github.com/gkadillak/gevent_socketio2

**Comment by shuoli84** [comment]
@gkadillak That's great.
